tests/ui/redundant_locals.rs

```diff
@@ -0,0 +1,111 @@
+//@aux-build:proc_macros.rs
```

Comment:
```suggestion
//@aux-build:proc_macros.rs:proc-macro
```
This needs the crate type now.

Reply:
Thanks, I had more or less given up.